arisia-remote/app/arisia/controllers/LoginController.scala

```diff
@@ -38,8 +40,9 @@ class LoginController (
           loginService.getPermissions(user.id).map { permissions =>
             val allowed =
               if (earlyAccessOnly)
-              // We're in early-access mode, so the general public is *not* allowed in:
-              permissions.superAdmin || permissions.admin || permissions.earlyAccess
+              // We're in early-access mode, so the general public is *not* allowed in
+              // In a dev environment, add your login to `arisia.allow.logins` in order to permit your own login:
+              permissions.superAdmin || permissions.admin || permissions.earlyAccess || allowedLogins.contains(user.id.v)
```

Comment:
Nothing Scala-specific, it just reflects the way the type is structured.  I have a standard concept here (in this system, not at the Scala level) of `StdString` -- basically, a strongly-typed wrapper around a String value.  (A poor man's version of Haskell newtype, if you've come across that.)  The `v` is the underlying String inside of there.

Really, `allowedLogins` should be a set of `LoginId` instead of String, so that I wouldn't have to dereference the String itself, and can properly use the strong type.  Hmm -- I might make that change...

Reply:
Yeah, changed accordingly -- thanks for pointing that out.